docs/features/logforwarding/outputs/elasticsearch-forwarding.adoc

```diff
@@ -94,31 +71,32 @@ spec:
 [source,yaml]
 ----
 kind: ClusterLogForwarder
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 metadata:
   name: instance
   namespace: openshift-logging
 spec:
   outputs:
     - name: external-es
       type: elasticsearch
-      elasticsearch: 
-        structuredTypeKey: kubernetes.namespace_name <1>
-        structuredTypeName: myParsedMessages <2>
-      url: 'https://<ES_URL>:9200'
+      elasticsearch:
+        url: 'https://example-elasticsearch-secure.com:9200'
+        version: 8
+        authentication:
+          username:
+            key: username
+            secretName: es-secret
+          password:
+            key: password
+            secretName: es-secret
+        index: "{.kubernetes.namespace_name|.log_type}" <1>
```

Comment:
The final piece to the template must be a static value, even though `log_type` should be present in all events. Also double pipe `||`.
```suggestion
        index: "{.kubernetes.namespace_name||.log_type||"missing"}" <1>
```